src/types/express.d.ts

```diff
@@ -148,6 +148,15 @@ type LawfulBasisSpecialPublicInterestStep = {
 
 type MoreOrganisationStep = string[];
 
+type FormStatus =
+  | "NOT STARTED"
+  | "IN PROGRESS"
+  | "CANNOT START YET"
+  | "NOT REQUIRED"
```

Comment:
These two are only relevant to particular steps on the form, not the form as a whole. So I think we can get rid of them.
```suggestion
```